specificatie/openapi.yaml

```diff
@@ -97,7 +97,7 @@ components:
                         Een aanduiding die aangeeft dat gegevens wel of niet verstrektmogen worden. Indien true: op verzoek van deze persoon is het verstrekken van gegevens over deze persoon aan bepaalde derden niet toegestaan.
         inOnderzoek:
           $ref: "#/components/schemas/ReisdocumentInOnderzoek"
-    ReisdocumentHal:
+    ReisdocumentHalBasis:
```

Comment:
ReisdocumentHal is hernoemd naar ReisdocumentHalBasis, maar ik zie geen nieuwe ReisdocumentHal. Komt dit in een volgende versie? Anders zie ik geen reden om dat te doen